consts.js

```diff
@@ -25,13 +25,14 @@ var thirteenStrings = [
     "PT", // PT is Brazilian political party represented by the number 13
     "Washington Luís", // Brazil's thirteenth president
     "Millard Fillmore", // Thirteenth President of the United States
+    "Louis XIII" // Thirteenth king of France
```

Comment:
```suggestion
    "Louis XIII", // Thirteenth king of France
```

Don't forget the comma! :)